.github/workflows/deploy.yaml

```
@@ -21,10 +18,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
+
+      - name: Az CLI login
+        uses: azure/login@v1
+        with:
+          client-id: 4a761bec-628d-4c4b-860a-4903cbecc963 #app registration Application ID
+          tenant-id: 3aa4a235-b6e2-48d5-9195-7fcf05b459b0
+          allow-no-subscriptions: true
+
+      - name: Get Azure principal token for Radix
+        run: |
+          token=$(az account get-access-token --resource 6dae42f8-4368-4678-94ff-3960e28e3630 --query=accessToken -otsv)
```

Where does the id come from? And shouldn't it be?:
```suggestion
          token=$(az account get-access-token --resource 4a761bec-628d-4c4b-860a-4903cbecc963 --query=accessToken -otsv)
```

---

From my understanding, this is a fixed application ID used to get a token, ref https://radix.equinor.com/guides/deploy-only/example-github-action-using-ad-service-principal-access-token.html

---

Mercury: https://github.com/equinor/mercury/pull/322/files

---

Never mind, mercury was wrong